src/test/resources/txt2.csv

```diff
@@ -0,0 +1,7 @@
+type,fruit,quantity,date
+s,banana,10,2020-10-17
+s,apple,5,2020-10-17
+b,banana,10,2020-10-15
+r,banana,2,2020-10-17
+b,apple,4,2020-10-16
+s,orange,1,2020-10-17
```

Comment:
add empty lines to the end of your csv files

Reply:
done